dev/source/docs/static-analysis.rst

```diff
@@ -0,0 +1,41 @@
+.. _static-analysis:
+
+===============
+Static Analysis
+===============
+
+Two tools are often used by ArduPilot to perform static analysis on the codebase, clang's "scan build" option and Coverity.
+
+----------------
+Clang Scan-Build
+----------------
+
+You can do your own static analysis using autotest:
+
+::
+
+   ./Tools/autotest/autotest.py clang-scan-build
+
+.. note::
+
+   You may need to install clang for this to work
+
+Afterwards you can view the results (e.g.):
+
+::
+
+   scan-view /home/pbarker/rc/ardupilot/tmp/scan-build-2020-08-11-114140-28761-1
```

Comment:
somehow make path generic?

Reply:
Suggested:
scan-view /home/<user>/<path to repo>/ardupilot/https://clang.llvm.org/get_started.html

Reply:
Adjusted it.